typescript/sdk/src/core/HyperlaneRelayer.ts

```diff
@@ -211,13 +211,25 @@ export class HyperlaneRelayer {
       }
 
       const dispatchReceipt = await event.getTransactionReceipt();
-      await this.relayMessage(dispatchReceipt, undefined, message);
+      const processReceipt = await this.relayMessage(
+        dispatchReceipt,
+        undefined,
+        message,
+      );
+
+      this.logger.info(
+        `Message ${message.id} was processed in ${
+          this.multiProvider.tryGetExplorerTxUrl(destination, {
+            hash: processReceipt.transactionHash,
+          }) ?? 'tx ' + processReceipt.transactionHash
+        }`,
+      );
```

Comment:
I would have expected the multi provider to already log this tbh

Reply:
it does not